app/__init__.py

```diff
@@ -2,6 +2,9 @@
 from .db import db, migrate
 from .models import task, goal
 import os
+from .routes.task_routes import tasks_bp
+from .routes.goal_routes import goals_bp
```

Comment:
Don't forget that the convention for naming blueprints is to name them `bp`. With each blueprint being named the same thing we will need to import them under an alias like so: 

```python
from .routes.task_routes import bp as tasks_bp
```